### PR TITLE
Add puppetlabs-strings initial support

### PR DESCRIPTION
**Pull request by jyaworski**
@rnelson0 @bastelfreak

### CODE REVIEW

**Comment by juniorsysadmin** [comment]
I *strongly* dislike PRs that sneak in more than one thing. Do a PR for the Ruby bumps (if they're available yet from rvm) then do this one.

File: moduleroot/.pmtignore [hunk 1]
```diff
@@ -0,0 +1 @@
+vendor/
```
Comment (juniorsysadmin) [inline moduleroot/.pmtignore]: Puppet docs say:
> If you have both a .pmtignore and a .gitignore file, the Puppet module tool will read the .pmtignore file over the .gitignore

My preference: stick with .gitignore

**Comment by jyaworski** [comment]
@juniorsysadmin done.

File: moduleroot/.pmtignore (hunk 1 above)
Comment (rnelson0) [inline moduleroot/.pmtignore]: FYI there are issues with PMT and it does not actually use .gitignore in many situations. Just something to keep in mind if we see module sizes exploding.

**Pull request merged**